Sources/MapboxCoreNavigation/Router.swift

```diff
@@ -25,6 +25,16 @@ public struct IndexedRouteResponse {
      */
     public let routeIndex: Int
     
+    /**
+     Returns a route from the `routeResponse` under given `routeIndex` if possible.
+     */
+    public var selectedRoute: Route? {
```

Comment:
Just `route` is sufficiently clear, or maybe `currentRoute` for consistency with `RouteProgress.currentLeg`. “Selected” implies that there’s a selection action built into the SDK, but that’s still the responsibility of the developer in general.